scripts/generate_assets.py

```diff
@@ -0,0 +1,101 @@
+from birdclef.models.embedding.tilenet import TileNet
+import pandas as pd
+from birdclef.workflows import motif
+from pathlib import Path
+import IPython.display as ipd
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from sklearn.decomposition import PCA
+from sklearn.cluster import KMeans
+from sklearn.preprocessing import LabelEncoder
+from jinja2 import Environment, FileSystemLoader
+import webbrowser
+import click
+from ast import literal_eval
+import os
+
+
+@click.command()
+@click.option('--intra', default="brnowl", help='Singular species to perform intraclass inspection')
+@click.option('--inter', default=["brnowl", "skylar", "norcar"], help='Array of 3 species to perform interclass comparison')
+def main(intra, inter):
+    model = TileNet.load_from_checkpoint(
+        "../data/processed/model/2022-04-12-v4/"
+        "embedding.ckpt",
+        z_dim=64,
+    )
+
+    df = pd.read_parquet("../data/processed/2022-04-03-motif-consolidated.parquet")
+    df["species"] = df.source_name.apply(lambda x: x.split("/")[1])
+    df[["species"]].groupby("species").size().sort_values(ascending=False)
+
+    data = []
+    root = Path("../data/raw/birdclef-2022")
+    for row in df[df.species == intra].sample(50).itertuples():
+        y = motif.load_audio(root / row.source_name, int(row.motif_0), 5)
+        data.append(y)
+
+    z = data[0]
+    plt.plot(z)
+    os.makedirs('generated_resources', exist_ok=True)
```

Comment:
Make the default generated resource directory in `data/intermediate/generated_resources`, and make it an option. May be useful to also be useful to give resources a name under that directory too e.g. `exp-2022-05-16` which is passed into the command.